core/src/main/java/org/kiwix/kiwixmobile/core/bookmark/BookmarksActivity.java

```diff
@@ -96,6 +96,7 @@ public boolean onPrepareActionMode(ActionMode mode, Menu menu) {
     public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
       refreshAdapter = false;
       if (item.getItemId() == R.id.menu_context_delete) {
+        dialogShower.show(KiwixDialog.DeleteBookmarks.INSTANCE,(Function0<Unit>)() ->{
```

Comment:
the indentation should be updated, format this

Reply:
Done Sir